ocaml/xapi/xapi_periodic_scheduler_init.ml

```diff
@@ -73,6 +75,24 @@ let register ~__context =
       (fun __context -> Xapi_subject.update_all_subjects ~__context
     )
   in
+  let sync_ssh_status ~__context =
+    let self = Helpers.get_localhost ~__context in
+    let ssh_service = !Xapi_globs.ssh_service in
+    let status = Fe_systemctl.is_active ~service:ssh_service in
+    Db.Host.set_ssh_enabled ~__context ~self ~value:status ;
```

Comment:
This shouldn't be in xapi_periodic_scheduler_init.ml. This is code that synchronises the DB with the state of the world, which is done in dbsync_slave.ml.